### ISSUE TITLE
Incorrect START position in output when bed file is used as input

### ISSUE THREAD

**Issue by ManavalanG** [lgmgeo/AnnotSV#139]
For this record in input [example bed file](https://github.com/lgmgeo/AnnotSV/blob/master/share/doc/AnnotSV/Example/test.bed)

```
1	149041933	149209289	DEL		Benign
```

[AnnotSV's output](https://raw.githubusercontent.com/lgmgeo/AnnotSV/master/share/doc/AnnotSV/Example/test_GRCh37.annotated.example.tsv) is this (just showing first few columns of the `full` row): 

```
1_149041933_149209289_DEL_1	1	149041933	149209289	-167356	DEL		Benign	full	q21.2	..........
```

As the START position in bed file is zero indexed, shouldn't the START position in AnnotSV output be 149041934 (one-indexed) instead of 149041933?

**Comment by lgmgeo** [comment]
Indeed:
- in a BED file, the first base on the chromosome is numbered 0 (zero indexed).
- in a VCF file, the first base on the chromosome is numbered 1 (one-indexed).

With AnnotSV, I didn't consider this feature when using an input BED file. Thanks for the report.
this will be taken into account in a future version.

**Comment by lgmgeo** [comment]
Hi @ManavalanG,

Bug should be fixed with the new 3.3.1 version.

_Outputs:_
AnnotSV produces a tab-separated values file (tsv) and optionally a VCF file. Both output files are now 1-based, with inclusive-end.

BED input file:
```
1 149041933 149209289 DEL Benign
```

TSV output file:
```
1_149041934_149209289_DEL_1     1       149041934       149209289       -167355 DEL     Benign  NA      full
```

VCF output file:
```
1       149041934       .       T       <DEL>   .       PASS    AnnotSV_ID=1_149041934_149209289_DEL_1
```

Thank you for any feedback you can provide me on this subject,
Best,

Véronique
